family2family/build.gradle

```diff
@@ -26,7 +30,7 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
     compile("org.springframework.boot:spring-boot-starter-data-rest")
-    compile("org.springframework.security.oauth:spring-security-oauth2:${springOauthVersion}")
+    compile("org.springframework.security.oauth:spring-security-oauth2")
```

Comment:
This equivalent of "newest"? Only worry is breaking changes in the release that generates unsolicited work.